stroom-util/src/main/java/stroom/util/io/CompressionUtil.java

```diff
@@ -0,0 +1,29 @@
+package stroom.util.io;
+
+import org.apache.commons.compress.compressors.CompressorStreamFactory;
+
+import java.util.Set;
+
+public final class CompressionUtil {
+
+    private static final String SEPARATOR = ", ";
+    public static final String SUPPORTED_COMPRESSORS = CompressorStreamFactory.BROTLI + SEPARATOR +
+            CompressorStreamFactory.BZIP2 + SEPARATOR +
+            CompressorStreamFactory.GZIP + SEPARATOR +
+            CompressorStreamFactory.PACK200 + SEPARATOR +
+            CompressorStreamFactory.XZ + SEPARATOR +
+            CompressorStreamFactory.LZMA + SEPARATOR +
+            CompressorStreamFactory.SNAPPY_FRAMED + SEPARATOR +
+            CompressorStreamFactory.SNAPPY_RAW + SEPARATOR +
+            CompressorStreamFactory.Z + SEPARATOR +
+            CompressorStreamFactory.DEFLATE + SEPARATOR +
+            CompressorStreamFactory.DEFLATE64 + SEPARATOR +
+            CompressorStreamFactory.LZ4_BLOCK + SEPARATOR +
+            CompressorStreamFactory.LZ4_FRAMED + SEPARATOR +
+            CompressorStreamFactory.ZSTANDARD;
+
+    public static boolean isSupportedCompressor(String compressorName) {
+        Set<String> outputCompressors = new CompressorStreamFactory().getOutputStreamCompressorNames();
+        return outputCompressors.contains(compressorName);
+    }
+}
```

Comment:
Seems odd to have a supported list of codecs but then support anything that commons-compress can support.  How about 
```java
    private static final String SEPARATOR = ", ";
    public static final Set<String> SUPPORTED_COMPRESSORS = Set.of(
                    CompressorStreamFactory.BROTLI,
                    CompressorStreamFactory.BZIP2,
                    CompressorStreamFactory.GZIP,
                    CompressorStreamFactory.PACK200,
                    CompressorStreamFactory.XZ,
                    CompressorStreamFactory.LZMA,
                    CompressorStreamFactory.SNAPPY_FRAMED,
                    CompressorStreamFactory.SNAPPY_RAW,
                    CompressorStreamFactory.Z,
                    CompressorStreamFactory.DEFLATE,
                    CompressorStreamFactory.DEFLATE64,
                    CompressorStreamFactory.LZ4_BLOCK,
                    CompressorStreamFactory.LZ4_FRAMED,
                    CompressorStreamFactory.ZSTANDARD)
            .stream()
            .map(String::toLowerCase)
            .collect(Collectors.toSet());

    public static final String SUPPORTED_COMPRESSORS_STR = SUPPORTED_COMPRESSORS.stream()
            .sorted()
            .collect(Collectors.joining(SEPARATOR));

    public static boolean isSupportedCompressor(final String compressorName) {
        return !NullSafe.isBlankString(compressorName)
                && SUPPORTED_COMPRESSORS.contains(compressorName.toLowerCase());
    }
```